x-pack/plugins/task_manager/server/task.ts

```diff
@@ -127,6 +127,16 @@ export const taskDefinitionSchema = schema.object(
         min: 1,
       })
     ),
+    /**
+     * The maximum number tasks of this type that can be run concurrently per Kibana instance.
+     * Setting this value will force Task Manager to poll for this task type seperatly from other task types
+     * which can add significant load to the ES cluster, so please use this configuration only when absolutly necesery.
+     */
+    maxConcurrency: schema.maybe(
+      schema.number({
+        min: 0,
+      })
+    ),
```

Comment:
I chose to keep `0` as an option in case a TaskType wanted to disable the ability to enable their task in a specific Kibana instance via configuration.
This came up in the original Reporting discussions around preventing tasks from running in certain circumstances (for example, an Alerting only Kibana instance that can't run any other task type).

It seemed straight forward enough that enabling this here made sense to me- but open for discussion if anyone thinks this should have a _min_ of 1.

Reply:
Added 🙏